src/Akka.Persistence.SqlServer.Tests/DbUtils.cs

```diff
@@ -55,32 +50,42 @@ public static void Initialize(string connectionString)
 
         public static void Clean()
         {
-            var connectionBuilder = new SqlConnectionStringBuilder(ConnectionString);
-            var databaseName = connectionBuilder.InitialCatalog;
-            using (var conn = new SqlConnection(ConnectionString))
+            var oldDatabaseName = _builder.InitialCatalog;
+            var databaseName = $"akka_persistence_tests_{Guid.NewGuid()}";
+            _builder.InitialCatalog = databaseName;
```

Comment:
Create a new database for each unit tests to prevent cross-polination between unit tests